client/src/core/injected_build_constants.ts

```diff
@@ -24,4 +24,4 @@ export const cdn_url = typeof CDN_URL !== "undefined" && CDN_URL;
 
 export const local_ip = is_dev && typeof LOCAL_IP !== "undefined" && LOCAL_IP;
 
-export const services_feature_flag = false;
+export const services_feature_flag = true;
```

Comment:
Typo for this branch, or were you testing how some change applied to the service inventory?